build.assets/Dockerfile

```diff
@@ -148,15 +153,15 @@ RUN mkdir -p $RUSTUP_HOME && chmod a+w $RUSTUP_HOME && \
 
 # Install Rust using the ci user, as that is the user that
 # will run builds using the Rust toolchains we install here.
+# Cross-compilation targets are only installed on amd64, as
+# this image doesn't contain gcc-multilib.
 USER ci
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --profile minimal --default-toolchain $RUST_VERSION && \
     rustup --version && \
     cargo --version && \
     rustc --version && \
-    rustup component add --toolchain $RUST_VERSION-x86_64-unknown-linux-gnu rustfmt clippy && \
-    rustup target add i686-unknown-linux-gnu && \
-    rustup target add arm-unknown-linux-gnueabihf && \
-    rustup target add aarch64-unknown-linux-gnu && \
+    rustup component add rustfmt clippy && \
+    if [ "$BUILDARCH" = "amd64" ]; then rustup target add i686-unknown-linux-gnu arm-unknown-linux-gnueabihf aarch64-unknown-linux-gnu; fi && \
```

Comment:
Do you need to `echo` the `rustup target add` command like you do for `gcc-multilib`?

Reply:
@zmb3 No, this is just an if statement. It calls `rustup` if value is true. The `echo` in `apt-get` is needed because there the `if` is inside backtick, so `echo` will just modify the list before the `apt-get` is executed.